lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,5 @@
+module RakeArguments
+  def self.flags
+    ARGV.drop_while { |e| e != '--' }.drop(1)
```

Comment:
Now we have a module, with one method of one line it might be time to consider inlining it again.

Rakefile line 17:
```
flags = ARGV.drop_while { |e| e != '--' }.drop(1)
```

I like it much better when the outer level is the only thing accessing the global/environment variables.